File: src/main/java/com/studybuddy/habitTracker/Habit.java
package com.studybuddy.habitTracker;

import com.studybuddy.user.User;
import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;


import java.util.ArrayList;
import java.util.List;

@Getter
@Setter
@Entity
public class Habit {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    private String name;
    private String description;
    private boolean monday;
    private boolean tuesday;
    private boolean wednesday;
    private boolean thursday;
    private boolean friday;
    private boolean saturday;
    private boolean sunday;




    public Habit (String name){
        this.name = name;

    }
    public Habit (){

    }


}

File: a-projekt/src/app/habittracker/habit.ts
export interface Habit {
  id: number;
  name: string;
  description: string;
  monday: boolean;
  tuesday: boolean;
  wednesday: boolean;
  thursday: boolean;
  friday: boolean;
  saturday: boolean;
  sunday: boolean;
  user: {
    id: number;
  };
}
